src/engine/openspaceengine.cpp

```diff
@@ -694,10 +689,10 @@ void OpenSpaceEngine::scheduleLoadSingleAsset(std::string assetPath) {
     _scheduledAssetPathToLoad = std::move(assetPath);
 }
 
-void OpenSpaceEngine::loadSingleAsset(const std::string& assetPath) {
+void OpenSpaceEngine::loadAsset_init(const std::string assetName) {
```

Comment:
The `_postInit` version of this function is not called anymore, meaning that the specification is not necessary anymore.   -> loadAsset

Also:  I'd change `const std::string` to `const std::string&` as you are only referencing the asset name in here when you are passing it on to the assetmanager

Reply:
Made these changes